test/CMakeLists.txt

```diff
@@ -49,6 +49,22 @@ if (BUILD_STATIC_LIBS)
   endif ()
   H5_SET_LIB_OPTIONS (${HDF5_TEST_LIB_TARGET} ${HDF5_TEST_LIB_NAME} STATIC 0)
   set_target_properties (${HDF5_TEST_LIB_TARGET} PROPERTIES FOLDER libraries/test)
+
+  if (HDF5_EXPORTED_TARGETS)
```

Comment:
@byrnHDF I just realized for the purposes of the API tests I was unconditionally installing the test lib here and below. Would it be appropriate to surround it with `if (HDF5_TEST_API_INSTALL)` or should we have something else since there may be other reasons to install the test lib?